hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh

```diff
@@ -159,8 +159,10 @@ if [ $(ps -ef|grep -v grep| grep java|grep -cE ${CONF}) -ne 0 ]; then
 fi
 echo "Starting HugeGraphPDServer..."
 
+JVM_OPTIONS="-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager"
```

Comment:
```suggestion
JVM_OPTIONS="-Dlog4j.configurationFile=${CONF}/log4j2.xml -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager"
```

Reply:
I found that pd here did not correctly configure the log4j file path...

Reply:
> I found that pd here did not correctly configure the log4j file path...

Actually, configure the log4j file path is not displayed here. It is actually configured in application.xml
logging:
config: 'file:./conf/log4j2.xml',

Reply:
@VGalaxies  Do we need to unify?

Reply:
Currently, for store, it is specified in both JVM_OPTIONS and application.xml, so it might be good to unify that.